src/vs/workbench/contrib/editSessions/common/editSessionsLogService.ts

```diff
@@ -17,7 +18,7 @@ export class EditSessionsLogService extends AbstractLogger implements IEditSessi
 		@IEnvironmentService environmentService: IEnvironmentService
 	) {
 		super();
-		this.logger = this._register(loggerService.createLogger(environmentService.editSessionsLogResource, { name: 'editsessions' }));
+		this.logger = this._register(loggerService.createLogger(environmentService.editSessionsLogResource, { id: editSessionsLogId, name: localize('cloudChangesLog', "Cloud Changes") }));
```

Comment:
@joyceerhl you do not need to create output channel now. You can include all necessary info while creating the logger and system will create it automatically.